src/lib/Commands/TestCommand/index.js

```diff
@@ -281,9 +310,28 @@ class TestCommand extends AbstractCommand {
     this._info(c.blue('Using ') + testFile.type + c.blue(' test file ') + testFile.name);
 
     // read/process test code
-    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim();
-    this._codeProcessor.variables.__FILE__ = path.basename(testFile.path);
-    testCode = this._codeProcessor.process(testCode);
+    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim().replace(LINE_AND_FILE_REGEXP, "@{__");
+    let unsupportedValues = {}; // map of syntetic_value : original_value
+    var environmentVars = "";
+    for (var prop in process.env) {
+      if (prop !== BUILD_API_KEY_ENV_VAR) { //deny to access for BUILD_API_KEY_ENV_VAR
+        var propValue = process.env[prop].replace(new RegExp('\\\\', 'g'), "\\\\");
```

Comment:
1) win7 may have some env property with \ symbols. So \ wil be replaced with \\ for the correct using of the property:
CommonProgramFiles=C:\Program Files\Common Files
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
The code is 
        local myVar = "#{env:CommonProgramFiles}";
        myVar = "#{env:CommonProgramFiles(x86)}";
The result of impTest working is:
        local myVar = "C:\\Program Files\\Common Files";
        myVar = "C:\\Program Files (x86)\\Common Files";
2) I can't say why, but the `/\\\\/g` doesn't work on win7:
Including source "C:sersmitryppDataoaming
pm
ode_modulesmptestrcli/../impUnit/index.nut"
3) The goal is repalce \ . So it should be escaped in regexp -> \\ But the using \ in code should be escaped too, so -> \\\\